# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/VRParent.h
# Commit: 0a6cc70b3c31
# Full Hash: 0a6cc70b3c31b6ce28a5fbbbdfa7a434955953c8
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 2
# Description:
#   Bug 1550422 - P9. Sync preferences in VR process when they change. r=daoshengmu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31017
# ==============================================================================

diff -r d4cbcf4f1900 -r 0a6cc70b3c31 gfx/vr/ipc/VRParent.h
--- a/gfx/vr/ipc/VRParent.h	Sat May 25 10:12:45 2019 +0000
+++ b/gfx/vr/ipc/VRParent.h	Sat May 25 10:12:40 2019 +0000
@@ -42,6 +42,7 @@
   mozilla::ipc::IPCResult RecvNotifyVsync(const TimeStamp& vsyncTimestamp);
   mozilla::ipc::IPCResult RecvUpdatePref(const GfxPrefSetting& setting);
   mozilla::ipc::IPCResult RecvUpdateVar(const GfxVarUpdate& pref);
+  mozilla::ipc::IPCResult RecvPreferenceUpdate(const Pref& pref);
   mozilla::ipc::IPCResult RecvOpenVRControllerActionPathToVR(
       const nsCString& aPath);
   mozilla::ipc::IPCResult RecvOpenVRControllerManifestPathToVR(